_pages/en/it-picture-long-term.md

```diff
@@ -1,35 +1,44 @@
 # IT Vision Picture - Long Term
 
-This is a working document to help create a picture of what will IITB look in the medium term for ESDC.
+This is a working document to help create a picture of what will IITB (Innovation Information and Technology Branch) look in the long term for ESDC (Employment and Social Development Canada).
 
 In the following sections, we're trying to take the same end state and create "views" where different audiences would be looking at ESDC and see different things.
 
-- [Citizens](#citizens)
-- [Private Sector](#private-sector)
-- [IITB-ESDC](#iitb-esdc)
-- [Government Partners](#government-partners)
+- [Citizens](#Citizens)
+  - [Automatic Enrollment and Delivery](#Automatic-Enrollment-and-Delivery)
+  - [Self-Service](#Self-Service)
+  - [Service Centers and Outreach](#Service-Centers-and-Outreach)
+- [Private Sector](#Private-Sector)
+- [IITB-ESDC](#IITB-ESDC)
+  - [Modern Open Organization](#Modern-Open-Organization)
+  - [Strong Culture of Learning](#Strong-culture-of-learning)
 
 ## Citizens
 
-- [Automatic Enrollment and Delivery](#automatic-enrollment-and-delivery)
-- [Self-Service](#self-service)
-- [Service Centers](#service-centers)
-- [IoT Integration](#iot-integration)
+- Cyber security and privacy - citizens have a choice to share their personal data for convenience
+- Generational change: Citizens are used to deal with technologies, they do not face technical illiteracy like 2019.
 
 ### Automatic Enrollment and Delivery
 
 Enrollment and delivery of services is automated by default and wherever possible.
+These Canadians are benefiting seamlessly from GC services.
 GC Services are integrated with widely used third party platforms and services (Banks, CRA, Social media, Job boards, Internet of things, ..).
+Services are accessible anywhere, anytime, from any device, and from any partner.
 90% of citizens (33 300 000 people) have a trusted Canadian profile used to verify/validate identity.
 These citizens receive their benefits seamlessly throughout their lives as major life events trigger the delivery of benefits.
 
 ### Self-Service
 
 The GC (OneGC) Portal provides access to services and benefits (not just ESDC) in tell us once fashion.
+See the [GC Digital Operations Strategic Plan: 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html).
+
 Registering to the Portal will create or use the trusted Canadian profile and enable automatic enrollment and delivery in the future.
 Help for using the Portal is available directly on the Portal using instant messaging (text, voice or video) as well as by phone.
+Results in easier access to services through delivering services to the client through hands-off automated processes or supplemented by staff if required.
+
+TODO: Add an architecture view mapping our GC partners (e.g. TBS DXP to integrate with OneGC Portal, SSC for public cloud).
```

Comment:
I think the TODO here should be in the IITB-ESDC section.
TBS, OneGC, SSC are of no concerns to citizens. Unless we want to add an architecture with 3rd party partners outside of the federal government. The architecture view map would then be OneGC (representing all of GC's departments), the other jurisdictions (e.g. Digital Identity), and non-government agencies that the citizens deal with (e.g. banks, employers)

Reply:
Moved!  IITB-ESDC is where we need the most work!